klaviyo-newsletter

catgonza opened the pull request:
Shopify section block for inserting a klaviyo sign up form

catgonza committed:
klaviyo-newsletter

Shopify section block for inserting a klaviyo sign up form

mgkcloud commented:
please ensure klaviyo-newsletter is placed in its own folder to seperate blocks

also ensure file extension is .liquid

catgonza committed:
Update installation  

changed include to render as include is deprecated
replaced add to product-template for add to theme.liquid. More straight forward as sometimes the product-template can be hard to locate in the backend. Eg: product-template can consist of product-form, product content etc. Or product-template can be named differently

catgonza committed:
Rename klaviyo-newsletter

Placed in its own folder and added .liquid file extension

catgonza committed:
Rename klaviyo sign up form

catgonza committed:
Create installation

mgkcloud commented:
for sticky-add-to-cart please ensure rendering from theme.liquid does not introduce button onto non product pages -> may need to include css display none unless page template == type product

catgonza committed:
Update sticky-add-to-cart

Added logic to ensure sticky ATC button only appears on product pages
